### PR TITLE
Add mvlempyr

### PR DESCRIPTION
**Pull request by Soopyboo32**
This was pain

### CODE REVIEW

**Comment by Woundez** [comment]
> This was pain

sorry man

File: src/plugins/english/mvlempyr.ts [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+    return loadedCheerio('#chapter').html() || '';
+  }
+
+  async searchNovels(
+    searchTerm: string,
+    page: number,
+  ): Promise<Plugin.NovelItem[]> {
+    if (page > 1) return [];
+
+    const allNovels = await this.getAllNovels();
```
Comment (Soopyboo32) [inline src/plugins/english/mvlempyr.ts:161]: yes this is actually how their website does searching :skull:
(made it cache locally tho as i assume the list of novels wont be changed too often)

File: src/plugins/english/mvlempyr.ts [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+    return this._allNovels;
+  }
+
+  async loadAll() {
+    const ret: WPData[] = [];
+    let page = 1;
+    // eslint-disable-next-line no-constant-condition
+    while (true) {
+      const pages = await this.loadNPages(page, 5);
+      page += 5;
```
Comment (Soopyboo32) [inline src/plugins/english/mvlempyr.ts:187]: there is around 20 pages so it takes a bit, loading 5 at a time makes it load almost instantly tho

Comment (Soopyboo32) [reply src/plugins/english/mvlempyr.ts:187]: (cus the files are relatively small most the time is spent waiting on response not downloading response)

**Review by nyagami — APPROVED** [review]

**Pull request merged**